WordPress/Classes/ViewRelated/System/Floating Create Button/CreateButtonCoordinator.swift

```diff
@@ -9,7 +9,7 @@ import Gridicons
     }
 
     var button: FloatingActionButton = {
-        let button = FloatingActionButton(image: Gridicon.iconOfType(.create))
```

Comment:
As @mindgraffiti pointed out, line 73 and 78 of this file still have the old syntax, and need to be updated.

Reply:
Thanks!